files/usr/share/cinnamon/applets/sound@cinnamon.org/applet.js

```diff
@@ -1420,6 +1420,11 @@ MyApplet.prototype = {
             this._control["change_" + type](device);
         });
 
+        let bin = new St.Bin({ x_align: St.Align.END, style_class: 'menuitem-detail' });
```

Comment:
I believe that `.popup-inactive-menu-item` is a built-in cinnamon style class which should have the same style.